src/components/Settings/SettingsLogs/index.tsx

```diff
@@ -1,14 +1,72 @@
 import React from 'react';
+import useSWR from 'swr';
+import Error from '../../../pages/_error';
+import LoadingSpinner from '../../Common/LoadingSpinner';
+import {
+  FormattedDate,
+  FormattedTime,
+  useIntl,
+  defineMessages,
+} from 'react-intl';
 
-// We will localize this file when the complete version is released.
+const messages = defineMessages({
+  logs: 'Logs',
+  logsDescription:
+    'You can access your logs directly in stdout (container logs) or looking in /app/config/logs/overseerr.logs',
```

Comment:
We will want to have the same `<code>` tags in here like we did in the temporary message. If you only put the tags in here by themselves though, they will be just rendered out in the string.

You can see how this is done in the `RequestList` component. I apply <strong> tags in there.